db/migrate/20161206125911_create_admins.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAdmins < ActiveRecord::Migration[5.0]
+  def change
+    create_table :admins do |t|
+      t.string :name
+      t.string :email
```

Comment:
@abapolov are you sure that we could have any admin without an email in our database?